helm/druid/templates/broker/role.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.rbac.create }}
```

Comment:
Should this also be checking `.Values.broker.enabled`? Feels like disabling the broker should also disable these resources from being created.

Reply:
Fixed